pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.pkgproj

```diff
@@ -1,14 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  
   <PropertyGroup>
-    <PackageVersion>2.2.0</PackageVersion>
+    <PackageVersion>2.3.0</PackageVersion>
```

Comment:
We could just remove this special case if you wanted.

Reply:
You suggesting just take the default version from Packages.props by deleting the explicit version?

Reply:
Sure